views/question-table.pug

```diff
@@ -1,6 +1,7 @@
 table.table
   thead
     tr
+      th Shown?
```

Comment:
We could have a better name for this I think. maybe `Visible`

Reply:
Yes, this is more descriptive.